src/pollingLink.ts

```
@@ -0,0 +1,49 @@
+import {
+  Operation,
+  NextLink,
+  FetchResult,
+} from './types';
+
+import * as Observable from 'zen-observable';
+
+import {
+  ApolloLink,
+} from './link';
+
+
+export default class PollingLink extends ApolloLink {
+
+  private pollInterval: number;
+  private timer;
+
+  constructor(pollInterval?: number) {
+    super();
+    this.pollInterval = pollInterval || 0;
```

I'd say we should just throw if the poll interval is not set. `0` doesn't seem like a reasonable default.